tradeoffs.Rmd

```diff
@@ -0,0 +1,172 @@
+#  (PART) Conclusion {-}
+
+# Tradeoffs
+
+::: {.alert .alert-success} 
+This somewhat opinionated discussion was added by [Max Held](https://www.maxheld.de).
+Your mileage may vary.
+:::
+
+Test doubles such as mocks or fakes can help you isolate your tests from the state of external HTTP API.
+This can be a good thing, because it allows you to test your code against a rare responses, or when the API is not available.
```

Comment:
you can link to https://books.ropensci.org/http-testing/pkgs-testing-chapter.html#why-do-we-need-special-packages-for-http-testing `#pkgs-testing-chapter`

Reply:
done!